handlers/workspaces_test.go

```diff
@@ -1200,7 +1201,83 @@ func TestGetWorkspaceUsers(t *testing.T) {
 }
 
 func TestGetUserDropdownWorkspaces(t *testing.T) {
+	teardownSuite := SetupSuite(t)
+	defer teardownSuite(t)
+
+	db.TestDB.DeleteWorkSpaceAllData()
+
+	oHandler := NewWorkspaceHandler(db.TestDB)
+	oHandler.userHasAccess = db.TestDB.DeleteWorkSpaceUserAccessData
+
+	person := db.Person{
+		Uuid:        uuid.New().String(),
+		OwnerAlias:  "test-alias",
+		UniqueName:  "test-unique-name",
+		OwnerPubKey: "test-pubkey",
+		PriceToMeet: 0,
+		Description: "test-description",
+	}
+	person2 := db.Person{
+		Uuid:        uuid.New().String(),
+		OwnerAlias:  "test-alias2",
+		UniqueName:  "test-unique-name2",
+		OwnerPubKey: "test-pubkey2",
+		PriceToMeet: 0,
+		Description: "test-description2",
+	}
+	db.TestDB.CreateOrEditPerson(person)
+	db.TestDB.CreateOrEditPerson(person2)
+
+	workspace := db.Workspace{
+		Uuid:        uuid.New().String(),
+		Name:        "test-workspace" + uuid.New().String(),
+		OwnerPubKey: person2.OwnerPubKey,
+		Github:      "https://github.com/test",
+		Website:     "https://www.testwebsite.com",
+		Description: "test-description",
+	}
+	db.TestDB.CreateOrEditWorkspace(workspace)
+	workspace = db.TestDB.GetWorkspaceByUuid(workspace.Uuid)
+	ctx := context.WithValue(context.Background(), auth.ContextKey, workspace.OwnerPubKey)
 
+	roles := []db.WorkspaceUserRoles{
+		{WorkspaceUuid: workspace.Uuid, OwnerPubKey: person2.OwnerPubKey, Role: "ADD BOUNTY"},
+		{WorkspaceUuid: workspace.Uuid, OwnerPubKey: person2.OwnerPubKey, Role: "UPDATE BOUNTY"},
+		{WorkspaceUuid: workspace.Uuid, OwnerPubKey: person2.OwnerPubKey, Role: "DELETE BOUNTY"},
+		{WorkspaceUuid: workspace.Uuid, OwnerPubKey: person2.OwnerPubKey, Role: "PAY BOUNTY"},
```

Comment:
Add a "VIEW REPORT" role to the user, which will make the user have 5 roles, it will fix the userHasAccess error.

Reply:
@elraphty I try but i got same error:
![image](https://github.com/stakwork/sphinx-tribes/assets/160427254/df01a676-3010-4d4e-a3cf-f63fd7c923bb)

Reply:
@elraphty I have try but i got same error:
![image](https://github.com/stakwork/sphinx-tribes/assets/160427254/813fc551-bb36-4d22-bc86-412fa480f841)

Reply:
I dropped a comment